src/nrepl/jack-in-dependency-versions.ts

```diff
@@ -0,0 +1,225 @@
+import * as state from '../state';
+import * as vscode from 'vscode';
+import * as child from 'child_process';
+import * as path from 'path';
+import * as fs from 'fs';
+import { parseEdn } from '../../out/cljs-lib/cljs-lib';
+
+export type JackInDependencyKey = 'nrepl' | 'cider-nrepl' | 'cider/piggieback';
+
+export type JackInDependencyVersions = Partial<Record<JackInDependencyKey, string>>;
+
+const JACK_IN_DEPENDENCY_LIBRARIES: Record<JackInDependencyKey, string> = {
+  nrepl: 'nrepl/nrepl',
+  'cider-nrepl': 'cider/cider-nrepl',
+  'cider/piggieback': 'cider/piggieback',
+};
+
+const JACK_IN_DEPENDENCY_KEYS = Object.keys(
+  JACK_IN_DEPENDENCY_LIBRARIES
+) as JackInDependencyKey[];
+
+const JACK_IN_DEPENDENCY_DEFAULTS: Record<JackInDependencyKey, string> = {
+  'nrepl': '1.3.1',
+  'cider-nrepl': '0.55.4',
+  'cider/piggieback': '0.6.0',
```

Comment:
These versions can be bumped to the latest (or any other specific version), before we merge the PR

Reply:
Which are the latest now? I think we should use latest. And in the changelog we can put the versions used in Calva currently, so that someone who wants to configure those can easily find them.